packages/graphql/tests/integration/directives/fulltext.int.test.ts

```diff
@@ -570,7 +564,7 @@ describe("@fulltext directive", () => {
         const session = driver.session({ database: databaseName });
 
         const cypher = `
-            CALL db.indexes() yield
+            SHOW INDEXES yield
```

Comment:
Do we still have to support the 4.1 version?
I think that `SHOW INDEXES` is available only from the 4.2 
https://neo4j.com/docs/cypher-manual/4.2/deprecations-additions-removals-compatibility/

Reply:
Nope, only 4.3 onwards. 🙂

Reply:
Cool!